build.publishing.gradle

```diff
@@ -0,0 +1,82 @@
+ext.sonatypeUser = System.env.SONATYPE_USER as String
```

Comment:
Guess we could inline these four variables, i.e. instead of having `username = sonatypeUser` we could have `username = System.env.SONATYPE_USER` (maybe with the `as String` part to, not sure if would be needed or not).

Reply:
I would prefer to keep this at the top, for readability.
This helps to understand easily what is missing when setting up.